content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/cordapp-packaging.md

```diff
@@ -103,16 +109,19 @@ The gradle plugin builds the CPB. To create a CPB for a CPI:
 
 Corda validates that uploaded CPIs are signed with a trusted key. To trust your signing keys, upload them with these commands:
 
-1. Import the gradle plugin default key into Corda:
+1. Import the Gradle plugin default key into Corda:
     ```shell
-    curl --insecure -u admin:admin -X PUT -F alias="gradle-plugin-default-key" -F certificate=@gradle-plugin-default-key.pem https://localhost:8888/api/v1/certificates/codesigner
+    curl --insecure -u admin:admin -X PUT -F alias="gradle-plugin-default-key" -F certificate=@gradle-plugin-default-key.pem https://localhost:8888/api/v1/certificates/cluster/code-signer
     ```
 2. Export the signing key certificate from the key store:
     ```shell
     keytool -exportcert -rfc -alias "signing key 1" -keystore signingkeys.pfx -storepass "keystore password" -file signingkey1.pem
     ```
 3. Import the signing key into Corda:
     ```shell
-    curl --insecure -u admin:admin -X PUT -F alias="signingkey1-2022" -F certificate=@signingkey1.pem https://localhost:8888/api/v1/certificates/codesigner
+    curl --insecure -u admin:admin -X PUT -F alias="signingkey1-2022" -F certificate=@signingkey1.pem https://localhost:8888/api/v1/certificates/cluster/code-signer
     ```
-{{< note >}}Use an alias that will be unique over time. Consider how certificate expiry will require new certificates with the same x500 name as existing certificates and define a naming convention that covers that use case.{{< /note >}}
+
+{{< note >}}
+Use an alias that will be unique over time. Consider how certificate expiry will require new certificates with the same X.500 name as existing certificates, and define a naming convention that covers that use case.
```

Comment:
Consider how certificate expiry will require new certificates with the same X.500 name as existing certificates, and define a naming convention that covers that use case.
>
After consideration that certificate expiry will require new certificates with the same X.500 name as existing certificates, define a suitable naming convention that covers that use case.